tce-core/AddElement/index.vue

```diff
@@ -25,6 +25,8 @@ import get from 'lodash/get';
 import SelectElement from './SelectElement';
 import SelectWidth from './SelectWidth';
 
+const FIXED_WIDTH_TYPES = ['ACCORDION', 'ASSESSMENT', 'BREAK', 'CAROUSEL', 'TABLE'];
```

Comment:
This should not be hardcoded anymore (it should come from config; `ui.forceFullWidth`) 😉 In general, there shouldn't be element type specific handling (all of that should come from config; so if you notice other occurrences please address it).